### DISCUSSION TITLE
Windows Support?

### DISCUSSION THREAD

**Discussion by prorusnet** [alecbcs/hyprspace#4] [Ideas]
Hello, will there be windows 10 and android support?

**Comment by alecbcs** [comment]
Hi @prorusnet! I think writing support for Windows could totally be possible! It's just that at the moment I don't actually have any devices that run Windows so I'd be a bit nervous about releasing something without being able to test it first. If someone from the community who runs Windows would be interested in contributing I'd be happy to accept some pull requests/work with them to build out some support for Windows devices. (In short it's on my roadmap but I don't have a way of doing it yet.)

As for Android, I don't really have any experience writing Android/IOS apps. I think mobile apps would be a really fun addition, but it might be something to think about a little further down the road once we've ironed out the protocol and verified it's secure/not leaking any info. If anyone's got Android/IOS development experience and would be interested on helping build apps for Hyprspace let me know!

**Reply by alecbcs** [reply]
@prorusnet I figured this post might make a bit more sense as a discussion post rather than an issue? Just so that we can collect some ideas for features over here and then people can upvote them so I can see what to spend time on in the future.

**Reply by alecbcs** [reply]
@prorusnet I'd forgotten about this discussion but Windows support for Hyprspace was officially released in v0.2.2.
